src/ranking/effectiveness/channel.ts

```diff
@@ -70,7 +69,7 @@ export namespace TypeChannelScore {
     return type + '_' + channel;
   }
 
-  export function getScore(specM: SpecQueryModel, stats: Stats, opt: QueryConfig) {
+  export function getScore(specM: SpecQueryModel, opt: QueryConfig) {
```

Comment:
Methods in this files don't have `schema`.   So replace `stats` with `schema` instead of simply removing.